docker-compose.yml

```diff
@@ -1,22 +1,22 @@
-version: '3.7'
+version: '3.9'

```

Comment:
Create separate pr for these changes.

Reply:
@shiara02 I thought this is in a separate pr??